client/src/templates/Challenges/codeally/show.tsx

```diff
@@ -246,35 +246,14 @@ class ShowCodeAlly extends Component<ShowCodeAllyProps> {
       </LearnLayout>
     ) : (
       <Hotkeys
-        innerRef={(c: HTMLElement | null) => (this._container = c)}
+        innerRef={this._container}
         nextChallengePath={nextChallengePath}
         prevChallengePath={prevChallengePath}
       >
         <LearnLayout>
           <Helmet title={`${blockName}: ${title} | freeCodeCamp.org`} />
           <Grid>
-            {CODEALLY_DOWN && superBlock === SuperBlocks.RelationalDb && (
-              <Row>
-                <Col md={8} mdOffset={2} sm={10} smOffset={1} xs={12}>
-                  <Spacer size='medium' />
-                  <Alert bsStyle='danger'>
-                    <p>
-                      <Trans i18nKey='intro:misc-text.course-maintenance'>
-                        <a
-                          href='https://www.freecodecamp.org/news/how-to-run-freecodecamps-relational-databases-curriculum-using-docker-vscode-and-coderoad'
-                          rel='noreferrer'
-                          target='_blank'
-                        >
-                          placeholder
-                        </a>
-                      </Trans>
-                    </p>
-                    <Spacer size='small' />
-                    <p>{t('intro:misc-text.progress-wont-save')}</p>
-                  </Alert>
-                </Col>
-              </Row>
-            )}
+            {superBlock === SuperBlocks.RelationalDb && CodeAllyDown()}
```

Comment:
```suggestion
            {superBlock === SuperBlocks.RelationalDb && <CodeAllyDown />}
```
This makes it obvious it's a component.

Reply:
🤦‍♂️

Reply:
I knew the syntax looked weird, but could not, for the life of me figure out why. Just shows how long I have been away from front-end stuff.

Reply:
it doesn't only make it obvious it's a component, it makes react render it as JSX element and not a regular one.